### DISCUSSION TITLE
problem transferring with HD case

### DISCUSSION THREAD

**Discussion by eddbull** [aldostools/webMAN-MOD#1101] [General]
Good evening, before version 4.85 of HFW I was able to use my case to put hard drives via USB to transfer games to the PS3, but now in versions 4.90 and 4.91 I am having a problem, when connecting the hard drive with the case on the PS3, the PS3 recognizes the HD, but when transferring games, the PS3 disconnects the HD, causing the transfer to fail, it's not a problem with the HD or the case, do you have any idea what it could be?

**Comment by aldostools** [comment]
If it happens while playing (reading mode) it could be that the external HDD is entering to sleep mode.
Some external HDD have tools that let disable the sleep mode or change the interval.
webMAN MOD has an USB polling option that is disabled by default.
![image](https://github.com/user-attachments/assets/f77d94ad-894a-470b-a659-b0146bd8f817)


If it happens during read-write operations like you commented, there are some possible causes:
- defective USB port
- defective USB cable
- defective HDD controller of the external case
- defective HDD

My suggestion is that try replacing the case and USB cable or try a different HDD to check if its the disk or the case.

You also has the option to transfer the files through FTP

**Reply by eddbull** [reply]
I'll try here, thank you very much
